packages/@okta/vuepress-site/docs/guides/migrate-to-okta-bulk/main/index.md

```diff
@@ -1,22 +1,31 @@
 ---
-title: Bulk Migration with Credentials
+title: Bulk migration with credentials
+meta:
+  - name: description
+    content: Perform a bulk migration of users into Okta using the Okta API
+layout: Guides
 ---
 
-## Performing a migration using the Okta API
-
 As part of your plan and preparation to migrate your users to Okta, you gathered your source data into an intermediate staging area such as a secure local database or a CSV file. This guide shows you how to use that data to create users and groups in Okta using the [Okta Users API](/docs/reference/api/users/) and the [Okta Groups API](/docs/reference/api/groups/).
 
-### Prerequisites
+---
+
+**Learning outcomes**
+
+Perform a bulk migration of users into Okta using the Okta API.
 
-To use this guide, you need the following:
+**What you need**
 
 * An Okta developer org. (Don’t have one? [Create an org for free.](https://developer.okta.com/signup))
 * Postman client to run API requests. See [Use Postman with the Okta REST APIs](https://developer.okta.com/code/rest/) for information on setting up Postman.
 * Example or test source data to test user and group creation requests. (Do not use real user data when testing!)
+* [A plan for migrating existing users to Okta](/docs/guides/migrate-to-okta-prerequisites/).
 
-**Note:** The examples in this guide are presented using cURL commands. Postman can generate request code for a number of programming languages that can help with development.
+---
+
```

Comment:
Does this guide merit a "Sample code" section here? I think that devs might appreciate knowing where to find the request code on the page.